app/database/users/users.py

```
@@ -0,0 +1,193 @@
+        with conn.cursor() as cursor:
+            sql = "delete from users where username = ?"
+            cursor.execute(sql, (username,))
+            conn.commit()
+            return cursor.rowcount > 0
+
+
+def update_user_by_username(username: str, fields: dict) -> bool:
+    if not fields:
+        return False
+    set_clause = ", ".join(f"{k} = ?" for k in fields)
+    values = list(fields.values()) + [username]
```

Ruff (0.15.12)

[warning] Consider `[*list(fields.values()), username]` instead of concatenation

Replace with `[*list(fields.values()), username]`

(RUF005)